tests/datasets/test_datasets.py

```diff
@@ -18,7 +18,7 @@
 from argilla.client.sdk.commons.errors import ForbiddenApiError
 from argilla.datasets import TextClassificationSettings, TokenClassificationSettings
 from argilla.server.contexts import accounts
-from argilla.server.security.model import WorkspaceUserCreate
+from argilla.server.security.model import User, WorkspaceUserCreate
```

Comment:
This is not the correct import for the type-checking on `argilla_user`, it is `from argilla.server.models import User`